### ISSUE TITLE
CDRM2 problem--previously worked fine for me

### ISSUE THREAD

**Issue by hsg120** [datawhores/OF-Scraper#310]
**Describe the bug**
This worked great for me for ages, so I'm not sure what I did to break it. But now when I run the program I get an error about not being able to reach the CDM service. At first I assumed this was a big, but I haven't seen anyone else with this same problem.

All auth.json info is entered manually, and for whatever it's worth I cannot reach http://172.106.17.134:8080/wv when I put it into a browser.

I'm likely doing something stupid--just not sure what it is.

**Example**

    Run "ofscraper"

**Expected behavior**
Should work according to documentation.

**Screenshots/Logs**

>[jordan@TurboEncabulator ~]$ ofscraper
> [startvals.printStartValues:26]  config path: /home/your_username/.config/ofscraper/config.json                                                                                   >startvals.py:26
 >[startvals.printStartValues:27]  profile path: /home/your_username/.config/ofscraper/main_profile                                                                                 >startvals.py:27
 >[startvals.printStartValues:28]  log folder: /home/your_username/.config/ofscraper/logging                                                                                        >startvals.py:28
> Welcome to OF-Scraper Version 3.6.8
>Key Mode: cdrm2
>
>
>[paths.cleanup:62]  Cleaning up temp files                                                                                                                                            >paths.py:62
>                                                                                                                                                                                                  
>                                                                                                                                                                                                  
>`CDM` service cdrm2 timed out and seems to be down
>This may cause a lot of failed downloads
>Please confirm by checking the url:http://172.106.17.134:8080/wv
> Consider switching
>https://of-scraper.gitbook.io/of-scraper/cdm-options

**Desktop:**
    OS: Arch Linux
    Browser: Chromium
    Version: 3.6.8
    binary or python binary

**Comment by cjb900** [comment]
:man_facepalming: not to be rude here but did you consider switching the CDM option as the error states :thinking:

**Comment by hsg120** [comment]
Hi! Not rude at all--I understand why you are asking. And yes, I did!

However, have you looked at the instructions on how to dump your own CDM? It's...involved, lol! I have tried sitting down to do it a couple times, but failed. 

So, I thought I'd check the open issues here and see if this is something anyone else is facing. My assumption was that the CDM error I'm receiving was a result of a bad configuration on my end or something, but I can't reach http://172.106.17.134:8080/wv if it put it in a browser either. So I was curious if I am the only one, or if everyone does in fact just dump their own CDM manually.

If it's the latter, then I'll try to go through the guide again tomorrow and give it another shot.
